package.json

```diff
@@ -36,7 +36,7 @@
   "homepage": "https://github.com/lifion/lifion-kinesis#readme",
   "main": "lib/index.js",
   "engines": {
```

Comment:
Right now, if you are using not supported version on node (ex. v16.10.0) you can easily install node modules. Ideally there should be a warning (I am not sure why I don't get one). However, let change it so it won't allow to install node modules unless you are in a supported version.
Some resources to check:
https://www.stefanjudis.com/today-i-learned/prevent-npm-install-for-not-supported-node-js-versions/
https://stackoverflow.com/questions/29349684/how-can-i-specify-the-required-node-js-version-in-package-json

Reply:
Awesome, I thought we did get the warning message, but I will add the additional .npmrc to enforce the engine-strict rule. It does seem like it can't be done inside pacakge.json only.